src/glyph.ts

```diff
@@ -257,17 +257,23 @@ export class Glyph extends Element {
     y_pos: number,
     point: number,
     code: string,
-    options?: { category: string }
+    options?: { category?: string; scale?: number }
   ): GlyphMetrics {
     const data = Glyph.cache.lookup(code, options?.category);
     const metrics = data.metrics;
     if (data.point != -1) {
       point = data.point;
     }
 
-    const scale = (point * 72.0) / (metrics.font.getResolution() * 100.0);
```

Comment:
Can we add something like: `const custom_scale = options?scale ?? 1;` here and use that below?

Reply:
Done but using camelCase:  `const customScale = options?scale ?? 1;`